todo_app/tests/test_view_model.py

```diff
@@ -0,0 +1,108 @@
+from typing import Iterable
+from todo_app.ItemClass import Item
+from todo_app.debug import debug
+
+# import sys
+# sys.setrecursionlimit(5000)
+
+def test_view_model_todo_property():
+    context = 'test_view_model.py'
+    doing = 'test_view_model_todo_property'
+    #arrange
+    from todo_app.ViewModelClass import ViewModel
+    items = [ 
+        Item('someArbTestID1','ATodoTask','64ae6f476f946f8c451a5cb9'),
+        Item('someArbTestID2','AnotherTodoTask','TodoID1'),
+        Item('someArbTestID3','ADoinTask','64ae6f476f946f8c451a5cba'),
+        Item('someArbTestID4','AnotherDoinTask','DoinID1'),
+        Item('someArbTestID5','ADoneTask','64ae6f476f946f8c451a5cbb'),
+        Item('someArbTestID6','AnotherDoneTask','DoneID1')
+    ]   
+    debug(context, doing, 'items',items)
+
+    # #act
+    item_view_model = ViewModel(items).todo_items
```

Comment:
This is a really confusing variable name - this isn't your view model, it's the `todo_items` list specifically! This might seem minor, but it left me quite confused with the asserts below